### DISCUSSION TITLE
Windows agent memory use since 4.7.0

### DISCUSSION THREAD

**Discussion by rmcclint** [wazuh/wazuh#21553] [Q&A]
We use Wazuh primarily for FIM. We have been having performance problems linked to memory utilization on our servers with the windows agent installed. It started happening when we updated to 4.7.0.  We have observed that each time, wazuh-agent.exe is consuming all of the paged pool memory it can get. Updated some servers to 4.7.2, but it still happens.  Restarting the service results in two processes, the old one slowly releasing its allocation and eventually terminating.  
I've searched thru release notes and this forum but I don't see any related topic.  
Does anyone know of a related issue or a way to further the diagnosis?

**Comment by Boo0b** [comment]
Exactly the same issue on many of my windows servers (2016&2019), Wazhu fixed a few memory leaking issues on previous releases https://github.com/wazuh/wazuh/issues/19580, but to me it did really solve the issue entirely.
